algorithms/ml/nearest_neighbor.py

```diff
@@ -0,0 +1,41 @@
+import numpy
+
+def distance(x,y):
+    """[summary]
+    HELPER-FUNCTION
+    calculates the (eulidean) distance between vector x and y.
+    
+    We use the numpy libriary
+
+    Arguments:
+        x {[tuple]} -- [vector]
+        y {[tuple]} -- [vector]
+    """
+    assert len(x) == len(y), "The vector must have same length"
+    import math
```

Comment:
why import math?

Reply:
@goswami-rahul for the function ```sqrt(...)``` that I used before ```numpy```.